docs/source/models/cause.rst

```diff
@@ -41,29 +41,251 @@ Common data sources for cause models
+will transition from a susceptible state to an infected state within a given timestep.** 
+
+	For example, with a timestep of one year, the probability that a simulant will transition from a 
+	susceptible (without MS) cause model state to an infected (with MS) cause model state is 2.8*10^(-5).
+
+**A Few Considerations for Incidence Data Sources:**
+
+As mentioned above, the denominator for incidence is person-years of the *at-risk* population, or the population 
+*without* condition (``1 - condition prevalence``). However, in certain scenarios, this may not always be the case.
+
+	When the prevalence of a condition is *small*, ``1 - prevalence`` will *approximately* equal ``1``. In 
+	these cases, incidence may be calculated as the number of new cases per person-years in the *entire* 
+	population as an approximation of a number of new cases per person-years in the *at-risk* population. 
+	This approximation will likely not have a large impact when condition prevalence is low, although the 
+	use and impact of this approximation in incidence data sources used for cause models should be considered.
+
+Further, it is important to consider that cause models are *state*-specific and not necessarily 
+*disease*-specific. What does this mean?
+
+	Consider a cause model in which an individual can trasition from a susceptible state to a mild disease 
+	state OR from a susceptible state to a severe disease state.
+
+	In this case, the incidence rate for overall disease (mild and severe) does not help us estimate the 
+	transition rates from susceptible to mild disease or to severe disease. In these cases, incidence rates 
+	specific to mild and severe disease are needed to inform the specific transitions present in the cause model.
+
+Lastly, see the section on hazard rates in non-standard data sources below to determine when hazard rates may be 
+preferrable to annual incidence rates as a data source for cause model transition rates.
+
+Remission Rates
++++++++++++++++
+
+Remission rates are defined as the **number of newly recovered cases from a condition that occur per person-year 
+of the population with the condition.**
+
+	For example, the remission rate of type II diabetes in the United States is 2.4 per 10,000 person-years.
+
+	This suggests that if we followed 10,000 individuals with type II diabetes for one year each, we would 
+	expect to see 2.4 individuals recover from type II diabetes.
+
+Remission rates can be used to estimate cause model transition rates and represent the probability that a 
+simulant in an infected (with condition) state will transition to a non-infected (without condition) state. 
+
+	For example, with a time step of one year, the probability that a simulant in the infected state in a 
+	model of type II diabetes in the United States will transition to a susceptible or recovered state 
+	within a timestep is 2.4*10^(-4), or 0.024%.
```

Comment:
Same comment as above in the incidence rate section about my confusion---is 0.024% precise or is it an approximation of `1 - exp(2.4e-4)`.  Not a big deal in this case, because they are really similar values.  But it could be important for getting the concept across clearly.